code/modules/mob/living/simple_animal/hostile/terror_spiders/purple.dm

```diff
@@ -49,7 +49,7 @@
 	. = ..()
 	if(.) // if mob is NOT dead
 		if(!degenerate && spider_myqueen)
-			if(dcheck_counter >= 10)
+			if(dcheck_counter >= 5)
```

Comment:
can just do `times_fired % 5 == 0`

Reply:
That isn't better. It is longer, and less clear.

Reply:
It will remove the `dcheck_counter` variable thus removing this if statement from 4 lines to 2 lines.

Reply:
Ah I did not notice you mentioned times_fired. That makes sense.